README.md

```diff
@@ -28,12 +28,12 @@ FastEstimator is a high-level deep learning library built on TensorFlow2 and PyT
 * Install PyTorch
     * CPU:
         ```bash
-        pip install torch==1.10.1+cpu torchvision==0.11.2+cpu torchaudio==0.10.1 -f https://download.pytorch.org/whl/torch_stable.html
+        pip install torch==1.7.1 torchvision==0.8.2 torchaudio==0.7.2
```

Comment:
Do we have to update tensorflow version as well in Prerequisites and Install TensorFlow?